src/views/lighttable.c

```diff
@@ -1264,7 +1264,7 @@ int expose_full_preview(dt_view_t *self, cairo_t *cr, int32_t width, int32_t hei
     dt_mipmap_cache_get(darktable.mipmap_cache, NULL, preload_stack[count], mip, DT_MIPMAP_PREFETCH, 'r');
 
   lib->image_over = DT_VIEW_DESERT;
-  cairo_set_source_rgb(cr, .1, .1, .1);
+  cairo_set_source_rgb(cr, 0.2, 0.2, 0.2);
```

Comment:
@pedrocr what about my latest proposal? This is not changing the place of the metadata but it makes it possible to hide them (using the current option) and have them appear when overing in the zone. This does not loose space for the image and at the same time it let the user view the image without the overlay. Seems like the best options to me.